### PR TITLE
otakomu.jp - PC

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [x] Annoyances — pop-ups, cookie warnings, etc;
- [x] Filters maintenance.

### Enter the issue address

 http://otakomu.jp/archives/30420478.html

### Add your comment and screenshots

<details><summary>screenshot</summary>

</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: JapaneseFilter/sections/general_extensions.txt
Comment (Yuki2718) [inline JapaneseFilter/sections/general_extensions.txt:307]: You can merge with `blog.esuteru.com#$#.siteRanking-main { margin-left: 0!important; }` and replace the comment `! https://github.com/AdguardTeam/AdguardFilters/issues/34774` with `! https://github.com/AdguardTeam/AdguardFilters/pull/131231`, they are essentially the same site. `blog.esuteru.com#$#.siteContent { margin-top: 0!important; }` and `otakomu.jp#$#.jackFix + div.siteContent { margin-top: 0 !important; }` can also be merged as `blog.esuteru.com,otakomu.jp#$#.jackFix + div.siteContent { margin-top: 0 !important; }`

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/general_extensions.txt
Comment (Yuki2718) [reply JapaneseFilter/sections/general_extensions.txt:307]: All the rest are fine, good work!

Comment (Alex-302) [reply JapaneseFilter/sections/general_extensions.txt:307]: will merge the rules after merging PR

**Review by Alex-302 — APPROVED** [review]

File: JapaneseFilter/sections/general_extensions.txt
Comment (SKEIDs) [reply JapaneseFilter/sections/general_extensions.txt:307]: thank you

**Pull request merged**
